devicetypes/smartthings/zwave-dimmer-switch-generic.src/zwave-dimmer-switch-generic.groovy

```diff
@@ -51,6 +51,7 @@ metadata {
 		fingerprint mfr: "0312", prod: "FF00", model: "FF02", deviceJoinName: "Minoston Dimmer Switch" //Minoston Toggle Dimmer Switch
 		fingerprint mfr: "0312", prod: "AA00", model: "AA02", deviceJoinName: "Evalogik Dimmer Switch" //Evalogik Smart Dimmer Switch
 		fingerprint mfr: "0312", prod: "C000", model: "C002", deviceJoinName: "Evalogik Dimmer Switch" //Evalogik Smart Plug Dimmer
+        fingerprint mfr: "0312", prod: "0004", model: "EE02", deviceJoinName: "Minoston Dimmer Switch" //Minoston  ZW31S Dimmer 
```

Comment:
Please fix spacing (change 4 spaces to 1 tab) and add the Raw Description (it may be found at graph.api.smartthings.com-> my devices -> ) in a comment after Your fingerprint -> // Minoston ZW31S Dimmer, Raw Description ...